Modules/QuestieQuestLog.lua

```diff
@@ -0,0 +1,239 @@
+                    region:SetTexture(textures[which]);
+                    region:SetWidth(256);
+                    region:SetHeight(256);
+                    textures[which] = nil;
+                end
+            end
+        end
+
+        -- Add in the new ones
+        for name, path in pairs(textures) do
+            local yofs, xofs = name:match("^([A-Z][a-z]+)([A-Z][a-z]+)$");
+            xofs = xofs and xOffsets[xofs];
+            yofs = yofs and yOffsets[yofs];
+            if (xofs and yofs) then
+                local region = QuestLogFrame:CreateTexture(nil, "ARTWORK");
+                region:ClearAllPoints();
+                region:SetPoint("TOPLEFT", QuestLogFrame, "TOPLEFT", xofs, yofs);
+                region:SetWidth(256);
+                region:SetHeight(256);
+                region:SetTexture(path);
+            end
+        end
+
+        -- And do some tricks with the empty quest log textures...
+        local topOfs = 0.37;
+        local topH = 256 * (1 - topOfs);
+
+        local botCap = 0.83;
+        local botH = 128 *  botCap;
+
+        local xSize = 256 + 64;
+        local ySize = topH + botH;
+
+        local nxSize = QuestLogDetailScrollFrame:GetWidth() + 26;
+        local nySize = QuestLogDetailScrollFrame:GetHeight() + 8;
+
+        local function relocateEmpty(t, w, h, x, y)
+            local nx = x / xSize * nxSize - 10;
+            local ny = y / ySize * nySize + 8;
+            local nw = w / xSize * nxSize;
+            local nh = h / ySize * nySize;
+
+            t:SetWidth(nw);
+            t:SetHeight(nh);
+            t:ClearAllPoints();
+            t:SetPoint("TOPLEFT", QuestLogDetailScrollFrame, "TOPLEFT", nx, ny);
+        end
+
+        local txset = { EmptyQuestLogFrame:GetRegions(); }
+        for _, t in ipairs(txset) do
+            if (t:IsObjectType("Texture")) then
+                local p = t:GetTexture();
+                if (type(p) == "string") then
+                    p = p:match("-([^-]+)$");
+                    if (p) then
+                        if (p == "TopLeft") then
+                            t:SetTexCoord(0, 1, topOfs, 1);
+                            relocateEmpty(t, 256, topH, 0, 0);
+                        elseif (p == "TopRight") then
+                            t:SetTexCoord(0, 1, topOfs, 1);
+                            relocateEmpty(t, 64, topH, 256, 0);
+                        elseif (p == "BotLeft") then
+                            t:SetTexCoord(0, 1, 0, botCap);
+                            relocateEmpty(t, 256, botH, 0, -topH);
+                        elseif (p == "BotRight") then
+                            t:SetTexCoord(0, 1, 0, botCap);
+                            relocateEmpty(t, 64, botH, 256, -topH);
+                        else
+                            t:Hide();
+                        end
+                    end
+                end
+            end
+        end
+    end
```

Comment:
Could this be relocated into a separate function which explains "the magic" a bit more? 😄

Reply:
I added a block comment that describes a little better what I think is actually happening in this routine. I say "what I think is happening" because this is a piece of legacy code that is not made by me but the original author of the double wide questlog addon.